CMakeLists.txt

```diff
@@ -771,6 +772,24 @@ find_feature(soxr ${SOXR_FEATURE_TYPE} ${SOXR_FEATURE_PURPOSE} ${SOXR_FEATURE_DE
 
 find_feature(GSSAPI ${GSSAPI_FEATURE_TYPE} ${GSSAPI_FEATURE_PURPOSE} ${GSSAPI_FEATURE_DESCRIPTION})
 
+option(WITH_GSSAPI "Compile support for kerberos authentication. (EXPERIMENTAL)" OFF)
```

Comment:
If int the remarks below `#ifdef WITH_GSSAPI` can be removed, move this to `winpr`